test/e2e/framework/node/runtimeclass/runtimeclass.go

```diff
@@ -49,3 +54,25 @@ func NewRuntimeClassPod(runtimeClassName string) *v1.Pod {
 		},
 	}
 }
+
+// Check if test-handler is configured. e.g. configured in kube-up on GCE:
+// see https://github.com/kubernetes/kubernetes/blob/eb729620c522753bc7ae61fc2c7b7ea19d4aad2f/cluster/gce/gci/configure-helper.sh#L3069-L3076
+func NodeSupportsPreconfiguredRuntimeClassHandler(ctx context.Context, f *framework.Framework) error {
```

Comment:
this is super brittle for a conformance test, that may be used by who knows which new runtime that is not crio or containerd and not spawned with gci scripts

Reply:
or only runs on this `It` tests that are not conformance?

Reply:
I agree it's a bit brittle, but the goal is to remove the skip based on GCE, because the test is currently being skipped in node e2e even though the runtime handler is configured. Overall, I think it's a net improvement over skipping based on GCE... if the container runtime has it configured it is signal we can run the test.  

Additionally this makes the test run fine in kind for example, while today it is being skipped.


Regarding conformance, these tests are not run against conformance already due to what you mentioned (we can't add the test to conformance since the runtime class in the container runtime might be configured). 

The two tests are:

(e2e node test; not conformance) 

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/node/runtimeclass.go#L130-L133


(e2e cluster test; not conformance, but rather node feature)

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/common/node/runtimeclass.go#L83-L94